tests/spec/core/linter-rules/local-refs-exist-spec.js

```diff
@@ -51,4 +51,21 @@ describe("Core Linter Rule - 'local-refs-exist'", () => {
     const result = await rule.lint(config, doc);
     expect(result).toBeUndefined();
   });
+
+  it("handles named anchors", async () => {
+    doc.body.innerHTML = `
+      <a name="NAME">PASS</a>
+      <a href="#NAME">PASS</a>
+      <a href="#NAME-NOT-EXIST">FAIL</a>
+      <a href="#NAME-NOT-EXIST">FAIL</a>
```

Comment:
```suggestion
```
Can remove this.

Reply:
> Curious to know how how named anchors are being used. Can you link a document?

Typically in markdown documents. For example, https://raw.githubusercontent.com/IMSGlobal/caliper-spec/develop/guides/certification-guide.md

Reply:
Removed and removed check for 2 occurrences (occurrences already checked by first test in this rule).